<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Water Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="generalTab">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <attribute name="title">
         <string>Control</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="3">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Meter Log:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QPushButton" name="controlReadLogButton">
              <property name="text">
               <string>Read Log</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="controlClearLogButton">
              <property name="text">
               <string>Clear Log</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Gallons Today:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLCDNumber" name="controlGallonsTodayLCD">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Leak Cond Control:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QPushButton" name="controlReadLeakConditionButton">
              <property name="text">
               <string>Read Leak Cond</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="controlClearLeakConditionButton">
              <property name="text">
               <string>Clear Leak Cond</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Leak Condition:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QLabel" name="controlLeakConditionLabel">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>None</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="3" rowspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Valve Control:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="0">
             <widget class="QRadioButton" name="controlForceOpenButton">
              <property name="text">
               <string>Force Open</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="controlAutoModeButton">
              <property name="text">
               <string>Automatic</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="controlForceClosedButton">
              <property name="text">
               <string>Force Closed</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="controlReadValvePositionButton">
              <property name="text">
               <string>Read Valve Position</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Valve Position:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="controlValvePositionLabel">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>System Control:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QPushButton" name="controlResetSystemButton">
              <property name="text">
               <string>Reset System</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="advancedTab">
        <attribute name="title">
         <string>Database</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Database Path:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="databasePathLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="databaseNewDBButton">
             <property name="text">
              <string>New DB</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="databaseBrowseButton">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="databaseConnectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="databaseDisconnectButton">
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="consoleTab">
        <attribute name="title">
         <string>Console</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2">
          <widget class="QTextEdit" name="consoleTextDisplay"/>
         </item>
         <item row="1" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="consoleInputLine"/>
           </item>
           <item>
            <widget class="QPushButton" name="consoleSendButton">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="consoleSettingsButton">
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="consoleClearButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="consoleConnectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="consoleDisconnectButton">
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="plotTab">
        <attribute name="title">
         <string>Plot</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Time Period:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0" colspan="2">
               <widget class="QComboBox" name="plotPeriodComboBox">
                <item>
                 <property name="text">
                  <string>Past Hour</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Past 24 Hours</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Past 72 Hours</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Past Week</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Past Month</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Past Year</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Custom</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDateTimeEdit" name="plotEndTimeEdit"/>
              </item>
              <item row="1" column="1">
               <widget class="QDateTimeEdit" name="plotStartTimeEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="plotStartTimeLabel">
                <property name="text">
                 <string>From:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="plotEndTimeLabel">
                <property name="text">
                 <string>To:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="plotMarkersLayout">
             <property name="title">
              <string>Markers:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_13">
              <item row="0" column="0">
               <widget class="QCheckBox" name="plotValveActuationsCheckBox">
                <property name="text">
                 <string>Valve Actuations</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="plotLeakConditionsCheckBox">
                <property name="text">
                 <string>Leak Conditions</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string>Total Gallons:</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_15">
                <item row="0" column="0">
                 <widget class="QLCDNumber" name="plotTotalGallonsLCDNumber">
                  <property name="lineWidth">
                   <number>0</number>
                  </property>
                  <property name="smallDecimalPoint">
                   <bool>false</bool>
                  </property>
                  <property name="digitCount">
                   <number>6</number>
                  </property>
                  <property name="mode">
                   <enum>QLCDNumber::Dec</enum>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::Flat</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="plotRefreshButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Refresh</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCustomPlot" name="plotPlot" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionConsole"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
